z/find_qids.py

```
@@ -72,42 +70,76 @@ def dump_one(file, data):
+                printe.output(f"<<yellow>> {term}: {qid}")
+                results[term][qid] = {
+                    "score" : 2,
+                    "matched_label" : term,
+                }
+    # ---
+    dump_data()
+
 
+def get_qids_new(english_terms_new):
+    # ---
+    titles = list(english_terms_new)
```

Ruff (0.12.2)

Local variable `titles` is assigned to but never used

Remove assignment to unused variable `titles`

(F841)